Extensions/xkit_patches.js

```diff
@@ -166,6 +166,14 @@ XKit.extensions.xkit_patches = new Object({
 	patches: {
 		"7.9.0": function() {
 
+			let global = globalThis || window;
```

Comment:
can't use `globalThis` on Firefox 64 and below, which is important considering our userbase is generally update-averse - need to support down to Firefox 48 ideally or any pre-Quantum version if not

it'd also break Opera fwiw

![image](https://user-images.githubusercontent.com/28949509/69108978-3334cc80-0a6e-11ea-882e-332dcd51f199.png)